testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -59,10 +60,21 @@ const STATUS_RESP_POST_CONDITION: &str = "abort_by_post_condition";
 pub const PATH_MICROBLOCK_SUBMIT: &str = "new_microblocks";
 pub const PATH_MEMPOOL_TX_SUBMIT: &str = "new_mempool_tx";
 pub const PATH_MEMPOOL_TX_DROP: &str = "drop_mempool_tx";
+pub const PATH_MINED_BLOCK: &str = "mined_block";
 pub const PATH_BURN_BLOCK_SUBMIT: &str = "new_burn_block";
 pub const PATH_BLOCK_PROCESSED: &str = "new_block";
 pub const PATH_ATTACHMENT_PROCESSED: &str = "attachments/new";
 
+#[derive(Clone, Serialize, Deserialize)]
+pub struct MinedBlockEvent {
```

Comment:
Is this something the sidecar will use, or is this only meant for testing?  Fine either way; just curious.

Reply:
Probably not API operators, but it's something that could be useful down the road for miners (or running a mock miner for analysis) -- event observers could get all sorts of information about their miner this way. Right now, it's really only useful for testing, but I figured it could be made into something more generally useful at some point.